tests/runtime/test_dlopen.c

```diff
@@ -24,15 +24,21 @@
 #include <dlfcn.h>
 #include <stdio.h>
 
+#ifdef __APPLE__
+#define SHLIB_EXT "dylib"
+#else
+#define SHLIB_EXT "so"
+#endif
+
 int
 main (int argc, char **argv)
 {
   int ret = 0;
-  const char *libpocl = "$ORIGIN/../../lib/CL/libpocl.so";
+  const char *libpocl = "$ORIGIN/../../lib/CL/libpocl." SHLIB_EXT;
   char libdevice[4096] = "";
   if (argc > 1)
     snprintf (libdevice, sizeof (libdevice),
-              "$ORIGIN/../../lib/CL/devices/%s/libpocl-devices-%s.so", argv[1],
+              "$ORIGIN/../../lib/CL/devices/%s/libpocl-devices-%s." SHLIB_EXT, argv[1],
```

Comment:
According to [this build log](https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/319615/logs/21) (commit https://github.com/conda-forge/pocl-feedstock/pull/64/commits/ad0663b8f50fbbd894eb33850795b982b6a13d06 in https://github.com/conda-forge/pocl-feedstock/pull/64), the device libraries build as `.so`, while `libpocl` builds as `.dylib`. I don't know if that's intended, but it seems to be the status quo.
```
2021-05-19T15:17:48.1187180Z -- Installing: $PREFIX/lib/libpocl.dylib
2021-05-19T15:17:48.1230540Z -- Installing: $PREFIX/lib/pocl/libpocl-devices-basic.so
2021-05-19T15:17:48.2551510Z -- Installing: $PREFIX/lib/pocl/libpocl-devices-pthread.so
```

Reply:
Seems not intended?

Reply:
Since they are dlopened, it doesn't matter what the suffix is. I removed this change in the last commit.